src/Seq.App.Opsgenie/OpsgenieApp.cs

```diff
@@ -129,40 +140,69 @@ public void Dispose()
         public async Task OnAsync(Event<LogEventData> evt)
         {
             if (evt == null) throw new ArgumentNullException(nameof(evt));
+            var suppressDiff = DateTime.UtcNow - _lastAlert;
+            if (suppressDiff.TotalSeconds < _suppressionTime.TotalSeconds) return;
 
             var message = _generateMessage.Render(evt);
             var description = _generateDescription.Render(evt);
             var priority = ComputePriority(evt);
-            var tags = ComputeTags(evt);
+            var tags = ComputeTags(evt, _includeTags, _includeTagProperty, _tags);
```

Comment:
Any chance of an example (screenshot of config?) showing how tag configuration would work in practice with this change?

Reply:
Yeah I should be able to dig it up. The difference was that it was not handling a comma delimited string passed from a Seq alert, because it was always expecting an array. This allows either to work as expected.

Reply:
@nblumhardt An example is shown below.

If I configure the Notification property tags on an alert, the current (v1.0) version of Seq.App.OpsGenie will ignore the comma-delimited tags sent from the alert, and only send the configured tags on the app instance.

With the enhancement, my comma-delimited set of tags will be used, although I'm not sure that the configured app instance tags are being included (it should be combining the OpsGenie alert tags with the event tags), and the whole set of tags isn't being logged - I'll have to check that.

Alert config:
![image](https://user-images.githubusercontent.com/30714560/164128004-9743f206-097f-4776-9561-809c844e1f4a.png)

OpsGenie instance tags config:

![image](https://user-images.githubusercontent.com/30714560/164128115-e433fd83-c7e7-4182-b0d2-81b15c5169b0.png)

Reply:
@nblumhardt  This is a bit curious. The unit tests for tag calculation work correctly - if a static tag is configured in the OpsGenie instance and an event passes a comma-delimited set of tags, it correctly combines the static tag with the event tags.

However in the DEV instance, thanks to the Handlebars template I'm using, I can see in the Opsgenie alert that Tags was passed from the alert and shows within the alert description, but doesn't actually get passed to OpsGenie as a Tags property.  The static tag ("Test") does get passed and this is seen in the Seq logging as well. 

Handlebars template is 
`<p>{{AppName}} failed with the error {{Description}}<br /><br /></p><table>{{#each $Properties}}<tr><td><b>{{@key}}:</b></td><td>{{this}}</td></tr>{{/each}}</table><p>Alert created by Seq</p>`

and the outcome in OpsGenie is below. You can see the tag ("Test") just below the title, but you can see in the description that the Handlebars template has shown the Tags property from the event passed by Alerting.

![image](https://user-images.githubusercontent.com/30714560/164133340-995d1103-80f3-4963-a524-e265b07f4620.png)

Suggests a problem in the ComputeTags method, but I don't see it - can't reproduce with a test case that emulates what gets passed. This is complicated by the fact that I'm leaving my current job so won't have access to their OpsGenie for testing in future ☹️

Reply:
@nblumhardt  I've amended the test case to evaluate a calculated OpsgenieAlert, for passing either an array or a comma separated value, and I do think it's working as intended. I don't know what I was seeing on the OpsGenie side, but at the end of the day, if the calculated OpsGenieAlert has the tags, they will be passed to Opsgenie.